### DISCUSSION TITLE
How should I handle lazyvim.json?

### DISCUSSION THREAD

**Discussion by tommyalatalo** [LazyVim/LazyVim#2727] [Q&A] [answered]
I've noticed that when I start lazyvim it now generates a `lazyvim.json` file with contents like this:

```
❯ cat lazyvim.json
{
  "extras": [

  ],
  "news": {
    "NEWS.md": "2123"
  },
  "version": 2
}
```

I've tried to find documentation for this file, but I can't seem to find any. Am I supposed to commit this file to git the same as lazy-lock.json, or just ignore it?

**Comment by yusufaktepe** [comment, marked as answer]
It's safe to ignore. If you are using any `LazyExtras`, you can add it to the lazy spec like [this](https://github.com/LazyVim/starter/blob/741ff3aa70336abb6c76ee4c49815ae589a1b852/lua/config/lazy.lua#L13-L15) to make it permanent.

**Comment by dpetka2001** [comment]
It contains the Extras that you have enabled via `:LazyExtras` UI, so if you want to go to another pc and have the exact same configuration you can just copy over the file and LazyVim will enable the same Extras you had in your configuration. I personally use `LazyExtras` to enable/disable the Extras I want, because it's more convenient than editing `lazy.lua` itself. Also, keep in mind, that when editing `lazy.lua` you might come across a case when some extras have to be imported before some others. `LazyExtras` takes care of that order for you.
